layouts/index.html

```diff
@@ -219,8 +214,8 @@ <h2 class="mt-9 text-white"><i class="fa-light fa-gears"></i> RAPIDS Release Sel
                          detailed installation steps, and information about supported platforms. </p>
                 </div>
                 <div class="col-md-6 py-3">
-                    <h2 class="text-white"><i class="fa-light fa-gauge-circle-bolt"></i> Test Drive the new cuDF</h2>
-                    <p>Try out cuDF's new pandas Accelerator Mode, with a free required account, right now by <a
+                    <h2 class="text-white"><i class="fa-light fa-gauge-circle-bolt"></i> Test Drive cuDF</h2>
+                    <p>Try out cuDF's pandas Accelerator Mode, with a free required account, right now by <a
```

Comment:
Avoid the possessive here. I left two possible suggestions -- take your favorite one.

I think the "free required account" piece of the first one is a bit awkward, so I'd lean towards the second suggestion.
```suggestion
                    <p>Try out cuDF pandas Accelerator Mode, with a free required account, right now by <a
```
```suggestion
                    <p>Try cuDF pandas Accelerator Mode in a free GPU-enabled notebook environment by <a
```